.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+## What does it do?
+
+## Checklist before requesting a review
+
+- [ ] Run `bundle exec rubocop` to test for code style violations and recommendations
+- [ ] Add Unit Tests (aka `specs/*_spec.rb`) if applicable
+- [ ] Run `bundle exec rspec` to run the whole test suite and ensure all your tests pass
+- [ ] Make sure you added an entry in [the `CHANGELOG.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#trunk) to describe your changes under the approprioate subsection.
```

Comment:
I wonder, given one could be tempted to add a new `## x.y.z` section to add their new entry in the CHANGELOG (like you initially did [here](https://github.com/wordpress-mobile/release-toolkit/pull/433#discussion_r1039995522)), instead of adding their entry under `## Trunk` — and only decide which version number we should use at _release_ time, for SemVer reasons — if we should provide a bit more details to this last checkbox, to be extra clear and avoid confusion for new contributors?

Something like this, maybe? wdyt?

```suggestion
- [ ] Make sure you added an entry in [the `CHANGELOG.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#trunk) to describe your changes under the approprioate existing `###` subsection of the existing `## Trunk` section.
```

Reply:
Yep, make a lot of sense, as it wasn't initially obvious to me where to add the CHANGELOG entry.